src/Fixture/CompanyExtensionEnabler.php

```diff
@@ -187,7 +187,7 @@ private function getCompanyExtensionList(string $extension_type): array {
     $extension_list = [];
 
     $top_level_packages = $this->packageManager->getAll();
-    if ($this->isSutOnly) {
+    if ($this->isSutOnly && $this->sut->getPackageName() != 'acquia/drupal-recommended-project') {
```

Comment:
Obviously a hack, I don't know how to tell in this context whether the SUT is a project template